### PR TITLE
[Features] Add event for scroll down to tickets section.

### PR DESCRIPTION
**Pull request by ratchagarn**
**1. Objective reason**

Arrow down icon are missing click event.
Related issue [#3](https://github.com/reactbkk/2.0.0/issues/3) 

**2. Description of change**

- Add simple handler click event for scroll down to tickets section.

### CODE REVIEW

File: components/HeroUnit.js [hunk 1]
```diff
@@ -84,3 +84,7 @@ function HeroUnitFooter () {
     </div>
   )
 }
+
+function ForDownArrowIconHandler () {
```
Comment (turboza) [inline components/HeroUnit.js]: @ratchagarn By JavaScript convention, I think it's better to set the method name starting with small case krub.

**Comment by turboza** [comment]
It would be better to make the cursor change to pointer when hovering on the triangle, so users will know it's clickable.

**Comment by ratchagarn** [comment]
@turboza OK, Got it.

**Review by dtinth — COMMENTED** [review]
I have just reviewed #40, #42, and #48.

Overall, #42 results in the best experience. For example, there is an animation when hovering the down arrow. When clicking, there is a scrolling animation.

Therefore, I have to close this PR. Thanks for your efforts again, especially your contribution on the icons.

**Comment by ratchagarn** [comment]
@dtinth Got it. Thanks.

**Pull request closed without merging**